graphs/ent/graph.py

```diff
@@ -251,6 +250,75 @@ def generate(abs_fpath, fname, blob, chunks=__chunks__, ibytes=__ibytes_dict__,
 
                 # # Eval the space required to show the section names
                 title_gap = int(longest_section_name / 2) * '\n'
+
+            elif bp.type == 'ELF':
+
+                log.debug('Adding ELF customisations')
+
+                # # Entrypoint (EP) pointer and vline
+                phy_ep_pointer = bp.get_physical_from_elf_v(bp.get_elf_ep()) / nr_chunksize
+                log.debug('{}: {}'.format('Entrypoint', hex(bp.get_elf_ep())))
+
+                host.axvline(x=phy_ep_pointer, linestyle=':', c='r', zorder=zorder-1)
+                host.text(x=phy_ep_pointer, y=1.07, s='EP', rotation=45, va='bottom', ha='left')
+
+                longest_section_name = 0
+
+                # # Section vlines
+                printed_section_offset = -100000
+                for index, section in bp.sections():
+                    zorder -= 1
+                    section_name = safe_section_name(section.name, index)
+                    section_offset = section.offset / nr_chunksize
+
+                    log.debug('{}: {}'.format(section_name, hex(section.offset)))
+                    
+                    if section.name.startswith( ('.fini', '.gnu', '.note.') ):
+                        continue
+                                        
+                    if section_offset - printed_section_offset < 15:
+                        continue
+						
+                    printed_section_offset = section_offset  
+                    host.axvline(x=section_offset, linestyle='--', zorder=zorder)
+                    host.text(x=section_offset, y=1.07, s=section_name, rotation=45, va='bottom', ha='left')
+
+                    # # Get longest section name
+                    longest_section_name = len(section_name) if len(section_name) > longest_section_name else longest_section_name
+
+                # # Eval the space required to show the section names
+                title_gap = int(longest_section_name / 2) * '\n'
+      
+            elif bp.type == '_unk_exec':
```

Comment:
This section will never be hit as if the binary blob cannot be parsed it is returned from ```bp.type```  as None and will then be parsed as a blob